### DISCUSSION TITLE
How do I create a window in 3.0.0pre

### DISCUSSION THREAD

**Discussion by Blatman** [neutralinojs/neutralinojs#635] [Q&A] [answered]
Been trying the latest source and the Native API has altered a bit. Got most of it going again but the window.create is a problem - maybe missing a trick.
This code works fine with 2.8.0/1.5.0 but with 3.0.0/1.5.0(latest) I get a runtime error -
                Neutralino.window.create('/readme.html', {
                    title: 'ReadMe',
                    width: 1000,
                    height: 480,
                    hidden: false,
                    processArgs: '--window-id=W_README' 
                }).catch(function(err) {
console.log('errREADME', err);
                });
err result with 3.0.0 is - 
{code: "NE_RT_NATRTER", message: "Native method execution error occurred. Failed because of: std::exception"}

As far as I can see this pattern for the create is still correct.

**Comment by shalithasuranga** [comment, marked as answer]
Ah, it was fixed recently. Thanks for reporting this - noticed this issue because of this discussion ;). Hope you will `git pull` and check again

**Reply by Blatman** [reply]
Corker - works well!
